src/plugins/intel_cpu/src/mlas/thread_pool.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+#include <functional>
+#include "mlas.h"
+
+namespace ov {
+namespace cpu {
+class OVThreadPool : public ThreadPool {
```

Comment:
1. Please see this comment https://github.com/openvinotoolkit/mlas/pull/1#discussion_r1271873450
2. Maybe better to rename to MlasThreadPool or OVMlasThreadPool